funcx_endpoint/funcx_endpoint/endpoint/rabbit_mq/task_queue_subscriber.py

```diff
@@ -0,0 +1,402 @@
+import logging
+import multiprocessing
+from typing import Callable
+
+import pika
+
+logger = logging.getLogger(__name__)
+
+
+class TaskQueueSubscriber(multiprocessing.Process):
+    """The TaskQueueSubscriber is a direct rabbitMQ pipe subscriber that uses
+    the SelectConnection adaptor to enable performance consumption of messages
+    from the service
+
+    """
+
+    def __init__(
+        self,
+        pika_conn_params: pika.connection.Parameters,
+        external_queue: multiprocessing.Queue,
+        kill_event: multiprocessing.Event,
+        endpoint_uuid: str,
+        _on_message_test_hook: Callable = None,
+        exchange: str = "tasks",
```

Comment:
Can `exchange` become a class attr, as with the other case of effectively constant values being described as class attrs?

Reply:
Done